client/views/fields/checkbox-group.html

```diff
@@ -0,0 +1,13 @@
+<sb-include src="/views/fields/container.html">
+  <div ng-repeat="value in field.values">
```

Comment:
Should use the checkbox markup from bootstrap: http://getbootstrap.com/docs/3.3/css/#checkboxes-and-radios